docs/markdown/spline_intro.md

```diff
@@ -81,7 +85,7 @@ The resulting spline curve can then be computed as follows:
 
 .. math::
 
-   \mathbf{S}(u)=(1-u)^{2}\mathbf{P}\_{1}+2u (1-u)\mathbf{P}\_{2}+u^{2}\mathbf{P}\_{3}
+   \mathbf{S}(u)=(1-u)^{2}\mathbf{P}_{1}+2u (1-u)\mathbf{P}_{2}+u^{2}\mathbf{P}_{3}
```

Comment:
is it just me or did you get this too? maybe a space after {1}?
![Screenshot 2024-05-17 at 15 06 58](https://github.com/tataratat/splinepy/assets/47114801/7ff3dcd2-11e0-4a66-b586-d4430ab7640c)

Reply:
For me, I don't get these issues. For me it displays this:
<img width="1051" alt="Screenshot 2024-05-17 at 16 13 29" src="https://github.com/tataratat/splinepy/assets/81433605/25269eae-2150-4c3b-9a1a-103985b0b880">